README.md

```diff
@@ -43,7 +46,7 @@ jlpm run build
 
 Install the extension
 ```bash
-python -m pip install .
+python -m pip install .[dev]
```

Comment:
Can you add the "two ways" of installing the extension, please ?

(for users) To install the extension use `python -m pip install .`
(for developers) and to install the extension and run the tests locally user `python -m pip install .[dev]`

Reply:
Editting:
I see that the "users" way of doing things is by using the docker image.
Just add something like, if you want to run the tests locally (user the ... .[dev])
@Soap2G opinons ?

Reply:
> Editting: I see that the "users" way of doing things is by using the docker image. Just add something like, if you want to run the tests locally (user the ... .[dev])

The installation for users has the command to install the extension from PyPI (`pip install reana-jupyterlab`). It can also be added `pip install .` as well, but IMO I think it is unnecessary.

However, I think mentioning that [dev] installation is to run the tests is a good idea.

Reply:
> The installation for users has the command to install the extension from PyPI (pip install reana-jupyterlab). pip install . an also be added, but IMO I think it is unnecessary.

Right, I forgot about the `pip install reana-jupyterlab` :)

Then just mention the [dev] installation, and we are good to go!

Reply:
> Editting: I see that the "users" way of doing things is by using the docker image. Just add something like, if you want to run the tests locally (user the ... .[dev]) @Soap2G opinons ?

Agreed on adding the sentence.